internal/handler/user.go

```diff
@@ -0,0 +1,90 @@
+package handler
+
+import (
+	"errors"
+	userModel "github.com/Azzonya/gophermart/internal/domain/user"
```

Comment:
тут не только модель. Тут целый пакет, userService 
Если хочется импортировать именно модели, что правильно - нужно их вынести в другой пакет
Ну а тут возникает вопрос зачем тогда пакеты usecase если ручки могут напрямую обращаться в сервисы

```suggestion
	userService "github.com/Azzonya/gophermart/internal/domain/user"
```